autoblocks/_impl/tracer.py

```diff
@@ -133,17 +123,19 @@ def _send_event_unsafe(
         trace_id = trace_id or self._trace_id
         timestamp = timestamp or datetime.now(timezone.utc).isoformat()
 
-        req = self._client.post(
+        event_dict = {
+            "message": message,
+            "traceId": trace_id,
+            "timestamp": timestamp,
+            "properties": merged_properties,
+        }
+
+        req = await self._client.post(
             url=INGESTION_ENDPOINT,
-            json={
-                "message": message,
-                "traceId": trace_id,
-                "timestamp": timestamp,
-                "properties": merged_properties,
-            },
```

Comment:
we can probably switch back to inlining the json? any reason to pull it out into `event_dict`?